pkg/ddevapp/config.go

```diff
@@ -943,6 +943,13 @@ func (app *DdevApp) RenderComposeYAML() (string, error) {
 		extraWebContent = extraWebContent + "\nRUN npm install -g n"
 		extraWebContent = extraWebContent + fmt.Sprintf("\nRUN n install %s && ln -sf /usr/local/bin/node /usr/local/bin/nodejs", app.NodeJSVersion)
 	}
+	// If node version 18+ enable corepack for yarn to be determined by project
+	// See https://yarnpkg.com/blog/release/4.0
+	c, _ := semver.NewConstraint(">=18")
+	v, _ := semver.NewVersion(app.NodeJSVersion)
+	if c.Check(v) {
+		extraWebContent = extraWebContent + fmt.Sprintf("\nRUN corepack enable")
```

Comment:
No need for sprintf:
```suggestion
		extraWebContent = extraWebContent + "\nRUN corepack enable"
```